### DISCUSSION TITLE
Provide API documentation & External Token access within the tool to allow external clients to create threads via external clients

### DISCUSSION THREAD

**Discussion by acherla** [StreetLamb/tribe#127] [Ideas]
Add another tab section to the TEAM view to show API documentation for leveraging an API for external use.  This will allow external clients to easily understand how to create a chat thread -> receive/handle the response for external clients to integrate with Tribe.  Allowing token creation will allow external users/apps to easily leverage the Tribe API for creating their own external chatbots or conversational experiences outside of the Tribe application.

Additional Requirements:
1. Add the ability for multiple unique tokens to be made per team (User Defined Token Name -> token) that can be used to create a conversational thread with a TEAM that can be used by external clients.  This would allow multiple applications to interact with a single team through their own unique token that can be shared with those application teams.  
2. Add an additional tab that documents how to leverage the token for external client use (document a simple curl command using the TEAM ID as reference for making the call) as a new tab in the TEAM section

Documentation should include
1. The external API to call via example in markdown format, included as a new tab in the TEAM tabular widget

**Comment by StreetLamb** [comment]
Thanks for the suggestion, I'll try to tackle this

**Reply by StreetLamb** [reply]
Hi @acherla , pushed #137 which adds the ability to stream responses from your teams and get the thread messages from public API endpoints.

**Reply by acherla** [reply]
Thanks ill review it tonight
